aries-site/src/examples/tokens/ComponentStates.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+// eslint-disable-next-line import/no-unresolved
+import { global, components } from 'hpe-design-tokens/grommet';
+import { Box, Button, FormField, TextInput } from 'grommet';
+
+const HoverButton = styled(Button)`
+  background: ${components.hpe.button.primary.hover.background};
+`;
+
+const FocusButton = styled(Button)`
+  outline-color: ${props =>
+    props.theme.global.colors[
+      props.theme.global.colors[global.hpe.focusIndicator.outline.color]
+    ]};
+  outline-width: ${global.hpe.focusIndicator.outline.width};
+  outline-style: ${global.hpe.focusIndicator.outline.style};
+  outline-offset: ${global.hpe.focusIndicator.outlineOffset};
+`;
+
+// eslint-disable-next-line react/prop-types
+export const ButtonStates = ({ size, tabIndex }) => (
+  <Box direction="row" gap="small">
+    <Button label="Rest" tabIndex={tabIndex} size={size} primary />
+    <HoverButton label="Hover" tabIndex={tabIndex} size={size} primary />
```

Comment:
Styles seem like they are coming through off. e.g. hover, focus
![Screenshot 2025-02-10 at 4 10 27 PM](https://github.com/user-attachments/assets/cc62d297-50f9-45fc-8c10-459668f8efe8)

Reply:
I see this locally -- can you try reinstalling node_modules, running `yarn build` in `design-tokens` before launching `aries-site` locally?

![Screenshot 2025-02-10 at 3 41 26 PM](https://github.com/user-attachments/assets/eaba9193-a61f-451d-8913-7aaa3e42afbf)

Reply:
yarn build design-tokens resolved.